string name
---
uint8 status
float32 x
float32 y
uint8 SUCCESS = 0
uint8 ILLEGAL_NAME = 1
uint8 UNKNOWN_ERROR = 2